src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/GoogleSignupModalComponent.java

```diff
@@ -0,0 +1,101 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows;
+
+import com.wikia.webdriver.common.logging.Log;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import javax.annotation.Nullable;
+
+public class GoogleSignupModalComponent extends WikiBasePageObject {
+
+  @FindBy(css = "button[name='__CONFIRM__']")
+  private WebElement appTermsConfirmButton;
+
+  @FindBy(css = "#signupUsername")
+  private WebElement usernameField;
+
+  @FindBy(css = "#signupEmail")
+  private WebElement emailField;
+
+  @FindBy(css = "#signupSubmit")
+  private WebElement registerButton;
+
+  @FindBy(css = "#facebookRegistrationForm")
+  private WebElement facebookRegistrationForm;
+
+  @FindBy(css = "#signupBirthDate")
+  private WebElement birthdateContainer;
+
+  @FindBy(css = "input.birth-month")
+  private WebElement birthMonthField;
+
+  @FindBy(css = ".birth-day")
+  private WebElement birthDayField;
+
+  @FindBy(css = ".birth-month")
+  private WebElement birthYearField;
+
+  public GoogleSignupModalComponent() {
+    super();
+  }
+
+  public void acceptWikiaAppPolicyNoEmail() {
+
+    new WebDriverWait(driver, 10).until(new ExpectedCondition<Boolean>() {
+      @Nullable
+      @Override
+      public Boolean apply(WebDriver input) {
+        return input.getWindowHandles().size() > 1;
+      }
+    });
+
+    //switch to Google modal;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)